core/main/Cargo.toml

```diff
@@ -50,6 +50,12 @@ env-file-reader = "0.2.0"
 sd-notify = { version = "0.4.1", optional = true }
 exitcode = "1.1.2"
 rand = "0.8"
+url = "=2.5.0"
```

Comment:
url version "2.2.2"  is defined in Ripple workspace. We may upgrade that and add workspace dependency here.